docs/user-guide/cli-using-team-configuration-team-leaders.md

```diff
@@ -204,10 +205,49 @@ The following example illustrates that the settings access multiple services usi
 }
 ```
 
-### Access to LPARs that access services through one API Mediation Layer using token authentication
+### Access to LPARs that access services through one API Mediation Layer using certificate authentication
 
 Access LPARs containing multiple services through API Mediation Layer with certificate authentication
 
 ```
-The following example illustrates....
+{
+    "$schema": "./zowe.schema.json",
+    "profiles": {
+        "zosmf": {
+            "type": "zosmf",
+            "properties": {
+                "basePath": "api/v1"
+            }
+        },
+        "cics": {
+            "type": "cics",
+            "properties": {
+                "basePath": "api/v1/cics"
+            }
+        },
+        "db2": {
+            "type": "db2",
+            "properties": {
+                "basePath": "api/v1/db2"
+            }
+        },
+        "base": {
+            "type": "base",
+            "properties": {
+                "certFile": "./zowe-cert.pem",
+                "certKeyFile": "./zowe-cert-key.pem",
+                "host": "example.com",
+                "port": 7554,
```

Comment:
Can the port in the above example (line 189) also be changed to 7554 to match? The default port for z/OSMF is 443, but for API ML is 7554.